frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -864,6 +869,19 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
         )}
       </SectionBox>
 
+      {ephemeralContainers.length > 0 && (
+        <SectionBox title={t('glossary|Ephemeral Containers')}>
+          {ephemeralContainers.map((container: KubeContainer) => (
+            <ContainerInfo
+              key={`ephemeral_container_${container.name}`}
+              resource={resource}
+              container={container}
+              status={statuses[container.name]}
```

Comment:
need new method for get ephem container status, also needs a new commit for init containers